presto-spark-base/src/test/java/com/facebook/presto/spark/TestPrestoSparkQueryRunner.java

```diff
@@ -102,6 +102,17 @@ public void testTableWrite()
                 45000);
     }
 
+    @Test
+    public void testZeroFileCreatorForBucketedTable()
+    {
+        assertUpdate(
+                getSession(),
+                format("CREATE TABLE hive.hive_test.test_hive_orders_bucketed_join_zero_file WITH (bucketed_by=array['orderkey'], bucket_count=1024) AS " +
```

Comment:
It is very likely that all the buckets are present. I would recommend filtering out the data for a couple of buckets, and maybe also decrease the total bucket count to, say 10 to make the test run faster